app/serializers/menu_serializer.rb

```diff
@@ -2,4 +2,5 @@ class MenuSerializer < ActiveModel::Serializer
   attributes :id, :name
```

Comment:
@SophieSjoberg I had to revert the serializers to the previous as they stopped working when moved into another dir.